docs/en/ingest-management/release-notes/release-notes-8.17.asciidoc

```diff
@@ -0,0 +1,218 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.17.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.17.0 relnotes
+
+[[release-notes-8.17.0]]
+== {fleet} and {agent} 8.17.0
+
+Review important information about the {fleet} and {agent} 8.17.0 release.
+
+[discrete]
+[[breaking-changes-8.17.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+{agent}::
+* Install images for {agent} on Debian 10 systems are no longer being made available. Debian 11 is now supported instead. {agent-pull}5847[#5847]
+
+[discrete]
+[[new-features-8.17.0]]
+=== New features
+
+The 8.17.0 release Added the following new and notable features.
+
+{fleet}::
+* Expose advanced file logging configuration in the UI. {kibana-pull}200274[#200274]
+
+{agent}::
+* Add support for pre-existing Active Directory user for unprivileged mode. {agent-pull}5988[#5988] {agent-issue}4585[#4585]
+* Add support for the Filestream integration. This will enable migration from the custom log integration which is based on a log input that is planned for deprecation. https://github.com/elastic/integrations/pull/11332[#11332]
+
+[discrete]
+[[enhancements-8.17.0]]
+=== Enhancements
+
+{fleet}::
+* Filter the integrations/packages list shown in the UI depending on the `policy_templates_behavior` field. {kibana-pull}200605[#200605]
+* Add a `<type>@custom` component template to integrations index template's `composed_of` array. {kibana-pull}192731[#192731]
+
+{fleet-server}::
+* Update `elastic-agent-libs` to version `0.14.0`. {fleet-server-pull}4042[#4042]
+
+{agent}::
+* Enable persistence in the configuration provided with our OTel Collector distribution. {agent-pull}5549[#5549]
+* Add Filebeat as a receiver for {agent} running in OTel mode. {agent-pull}5833[#5833]
+* Restrict using the CLI to upgrade {fleet}-managed {agents}. With this implementation, upgrading a {fleet}-managed agent using the CLI is supported only if the agent is running in privileged mode and the user provides a `force` flag. {agent-pull}5864[#5864] {agent-issue}4890[#4890]
```

Comment:
```suggestion
* Restrict using the CLI to upgrade for {fleet}-managed {agents}. {agent-pull}5864[#5864] {agent-issue}4890[#4890]
```

We don't actually want people to do this and the `--force` flag is hidden as an escape hatch.